stdlib/2/whichdb.pyi

```diff
@@ -0,0 +1,8 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/whichdb.py
+
+from types import ModuleType
+from typing import Optional
+
+dbm = ...  # type: Optional[ModuleType]
+
+def whichdb(filename: str) -> Optional[str]: ...
```

Comment:
The return value can be `None` if we pass non-existent filename: https://hg.python.org/cpython/file/2.7/Lib/whichdb.py#l73